bundle/Templating/Twig/Extension/NetgenTagsRuntime.php

```diff
@@ -95,4 +95,24 @@ public function getContentTypeName($contentType)
 
         return $contentType->getName();
     }
+
+    /**
+     * @param \Netgen\TagsBundle\API\Repository\Values\Tags\Tag $tag
+     *
+     * @return \Netgen\TagsBundle\API\Repository\Values\Tags\Tag|null
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException
```

Comment:
You don't actually throw `UnauthorizedException` from this method, so no need to document it.

Reply:
Done in https://github.com/netgen/TagsBundle/commit/d9eaed7b6a0551430ece3c48e367a254251f7701